x-pack/plugins/security_solution/cypress/tasks/entity_analytics.ts

```diff
@@ -31,3 +36,16 @@ export const enableJob = () => {
 export const navigateToNextPage = () => {
   cy.get(ANOMALIES_TABLE_NEXT_PAGE_BUTTON).click();
 };
+
+export const riskEngineStatusChange = () => {
+  cy.get(RISK_SCORE_SWITCH).should('not.have.attr', 'disabled');
+  cy.get(RISK_SCORE_SWITCH).click();
+};
+
+export const updateRiskEngine = () => {
+  cy.get(RISK_SCORE_UDATE_BUTTON).click();
+};
+
+export const updateRiskEngineConfirm = () => {
+  cy.get(RISK_SCORE_UPDATE_CONFIRM).click();
```

Comment:
Why this action is wrapped in `updateRiskEngineConfirm` and not part of `updateRiskEngine`?

Reply:
It open modal, and there some ways to cancel this model, so I want to split those actions